Ensure and document OpenVDB support (#475)
Ekumen-OS/beluga: beluga_vdb/README.md

# Beluga VDB

<div align="center">
<img alt="Shows the Beluga logo." src="images/57fe338a-3e7e-46b3-ad2f-17011d6d306a.png" height="200">
</div>

## 🌐 Overview

> BelugaVDB is a library extension for `beluga` that integrates [OpenVDB](https://www.openvdb.org/), enabling advanced 3D localization capabilities. Currently, this extension uses the powerful volumetric data structure of OpenVDB to efficiently process maps and 3D pointcloud data, providing the localization of the autonomous system.

Reviewer: @pvela2017 meta nit: consider using less bells and whistles
```suggestion
BelugaVDB is a library extension for `beluga` that integrates [OpenVDB](https://www.openvdb.org/), enabling advanced 3D localization capabilities. Currently, this extension uses OpenVDB to efficiently process 3D maps and pointcloud data.
```

Reviewer: Perhaps we can point to the VDB mapping repository too.

Author: Added VDB mapping on [5cdc016](https://github.com/Ekumen-OS/beluga/pull/475/commits/5cdc0169d3c9306da9bab59b1b4cb600f7fe9489)

> BelugaVDB is a library extension for `beluga` that integrates [OpenVDB](https://www.openvdb.org/), enabling advanced 3D localization capabilities. Currently, this extension uses OpenVDB to efficiently process 3D maps and pointcloud data. Maps in `vdb` format can be generated using any third party library such as [VDB Mapping](https://github.com/fzi-forschungszentrum-informatik/vdb_mapping).

Reviewer: Also, any? What do we need of the map? Just active vs inactive voxels, right? We should leave a note about that.

> BelugaVDB is a library extension for `beluga` that integrates [OpenVDB](https://www.openvdb.org/), enabling advanced 3D localization capabilities. Currently, this extension uses OpenVDB to efficiently process 3D maps and pointcloud data. Maps in `vdb` format can be generated using any third party library such as [VDB Mapping](https://github.com/fzi-forschungszentrum-informatik/vdb_mapping).

Author: Good catch! I forgot we need to do some post-processing to the map generated by VDB Mapping. We need a level-set. I added some code on how to achieve this and how to visualize the map.

## 📦 External Dependencies

### OpenVDB

> For Ubuntu 24.04 and newer versions OpenVDB can be installed using `apt`:

Reviewer: @pvela2017 nit: 
```suggestion
For Ubuntu 24.04 and newer distributions, OpenVDB can be installed using `apt`:
```

Author: Changed on [5cdc016](https://github.com/Ekumen-OS/beluga/pull/475/commits/5cdc0169d3c9306da9bab59b1b4cb600f7fe9489)

```bash
sudo apt install libopenvdb-dev
```

> Older version must be isnstalled from sources as follow:

Reviewer: @pvela2017 nit: 
```suggestion
For older distributions, it must be installed from sources as follows:
```
